freezeyt/freezer.py

```diff
@@ -369,7 +401,8 @@ def handle_redirects(self):
         for task in self.redirecting_tasks.values():
             if task.redirects_to.status != TaskStatus.DONE:
                 raise InfiniteRedirection(
-                    f'{task.get_a_url()} redirects to {task.redirects_to.get_a_url()}, which was not frozen (most likely because of infinite redirection)'
+                    f'{task.get_a_url()} redirects to {task.redirects_to.get_a_url()},'
+                    + 'which was not frozen (most likely because of infinite redirection)'
```

Comment:
Pokročilá verze úkolu z první lekce :)
![image](https://user-images.githubusercontent.com/302922/126285933-4004fb60-8289-4d3d-b147-78e3efe71d40.png)

Reply:
Není to chyba, jen jsem chtěl zkrátit řádek přišlo mi to dlouhé...

Reply:
Ale chybí tam mezera po čárce.

Reply:
jo takhle... Já to pochopil, že to myslíš proč jsem to opravoval co tam bylo za chybu..